docs/event-bus/docs/013-service-programming-model.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Service Programming Model
+type: Details
+---
+# Service Programming Model
+
+You can configure a microservice deployed in Kyma to receive Events from the Event Bus by creating a [subscription custom resource](https://github.com/kyma-project/kyma/blob/master/resources/cluster-essentials/templates/eventing-subscription.crd.yaml). Find various configuration options in the [Consume Events](https://github.com/kyma-project/kyma/blob/master/docs/event-bus/docs/011-details-event-flow-requirements.md#consume-events) section.
+
+## Event Delivery
+
+The Event is delivered as an `HTTP POST` request. Event Metadata is a part of an HTTP request Headers. Event Payload is the body of the request.
+
+![TEST](assets/service-programming-model.png)
+
+1. The Event is published to the Kyma Event Bus from an external system instance e.g. `commerce` in a bound Remote Environment.
```

Comment:
@abbi-gaurav can you please review these 3 actions validity?

Reply:
The actions are valid.

Reply:
@klaudiagrz, kindly rephrase.

Reply:
I've already included this in my comments.